src/main/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepository.java

```diff
@@ -84,4 +84,9 @@ public EventType mapRow(ResultSet rs, int rowNum) throws SQLException {
                 "SELECT et_event_type_object FROM zn_data.event_type",
                 new EventTypeMapper());
     }
+
+    @Override
+    public void removeEventType(String name) {
+        jdbcTemplate.update("DELETE FROM zn_data.event_type WHERE et_name = ?", name);
```

Comment:
if it was possible to sql inject here, yes. Like ? been replaced by "anything OR 1=1" then we would erase the database :sweat_smile: but I think that jdbc does allow it.